default.nix

```diff
@@ -96,24 +98,68 @@ let
       ghc = pkgs.haskell.compiler.ghc843;
     };
 
+    playgroundGhc = pkgs.haskell.packages.ghc843.ghcWithPackages (ps: [
+      haskellPackages.plutus-playground-server
+      haskellPackages.plutus-playground-lib
+      haskellPackages.plutus-use-cases
+    ]);
+
     localPackages = localLib.getPackages {
       inherit (self) haskellPackages; filter = localLib.isPlutus;
     };
     tests = {
       shellcheck = pkgs.callPackage localLib.iohkNix.tests.shellcheck { inherit src; };
       hlint = pkgs.callPackage localLib.iohkNix.tests.hlint {
         inherit src;
-        projects = localLib.plutusPkgList;
+        projects = let
+                     fixPlaygroundServer = v: if v != "plutus-playground-server" then v else "plutus-playground/plutus-playground-server";
+                     fixPlaygroundLib = v: if v != "plutus-playground-lib" then v else "plutus-playground/plutus-playground-lib";
```

Comment:
This is pretty ugly. Can't they just live at the top-level?

Reply:
indeed it is and yes, they could do, would prefer some more opinions on this though. @j-mueller  @krisajenkins ?

Reply:
I think it's ugly too

Reply:
Arguably it's the fault of the interface to the hlint thing. I feel like it could be cleverer and take a derivation and use the source location or something.